app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -3628,6 +3629,8 @@ public boolean onLongClick(final View v) {
             fragmentListener.onMoreOptionsLongClicked();
             hideControls(0, 0);
             hideSystemUIIfNeeded();
+        } else if (v.getId() == binding.share.getId()) {
+            getUrlClicked();
```

Comment:
Just move the function inline, as it is small
```suggestion
        if (currentMetadata != null) {
            ShareUtils.copyToClipboard(context, currentMetadata.getMetadata().getOriginalUrl());
        }
```

Reply:
Done